### PR TITLE
Re-enable integration test for autoscaling_instance* modules

### LINKED ISSUE #2362: Re-enable integration test for autoscaling_instance* modules
### Summary

Re-enable integration test for autoscaling_instance* modules.

Tests were run locally and temporarily disabled due to missing permissions.

See also https://github.com/ansible-collections/amazon.aws/pull/2296

### Issue Type

Feature Idea

### Component Name

autoscaling_instance, autoscaling_instance_info

### Additional Information

```yaml (paste below)

```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

Fixes #2362 

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by gravesm** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: tests/integration/targets/autoscaling_instance/tasks/main.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Beware: most of our tests here are run in parallel.
 # To add new tests you'll need to add a new host to the inventory and a matching
 # '{{ inventory_hostname }}'.yml file in roles/ec2_asg/tasks/
+# testing
```
Comment (alinabuzachis) [inline tests/integration/targets/autoscaling_instance/tasks/main.yml:5]: We don't need this, right?

**Review by alinabuzachis — APPROVED** [review]

**Review by gravesm — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Review by tremble — APPROVED** [review]

**Comment by tremble** [comment]
@GomathiselviS Thanks for the follow-up on this (including the gentle reminders on https://github.com/mattclay/aws-terminator/pull/311)

**Pull request merged**